usaspending_api/api_contracts/contracts/v2/search/spending_by_geography.md

```diff
@@ -33,9 +33,7 @@ This endpoint takes award filters, and returns aggregated obligation amounts in
                 + `county`
                 + `district`
         + `geo_layer_filters` (required, array[string])
```

Comment:
I would almost like the example for this param.  
`Members
      Any abbreviated US state (e.g. MD, VA, DC, CA)`

Reply:
Well, MD, PA, ... are only applicable when `geolayer` = `state`. For `county` they are 5-digit strings like `31031`. We do need more descriptive examples/definitions, but this PR would explode and take all week if I attempted to address everything.

This is more of an example of bad API design rather than a short-coming of API Contracts.